CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules/)
 
+set(CMAKE_BUILD_TYPE "Debug")
```

Comment:
This probably also shouldn't be in the final version, right?

Reply:
Correct again!